content/parts/en/datasets.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: "Datasets"
+html_description: "Download datasets"
+introduction: "Introductory disclaimer text presenting the varying quality depending on maintainers of the collections"
+related_datasets:
+  title: "Related datasets"
+  datasets:
+    - website: https://privacypolicies.cs.princeton.edu/ghfront/
+      name: Princeton-Leuven Longitudinal Corpus of Privacy Policies
+      description: Historical website privacy policies spanning over two decades.
+    - website: https://www.usableprivacy.org/data
+      name: Usable Privacy Policy
+      description: Curated list of data and tools.
```

Comment:
This makes us a meta-list… Could we instead list the datasets listed there? 🙂 

```yaml
  - website: https://www.usableprivacy.org/data
    name: OPP-115 Corpus (ACL 2016)
    description: Collection de politiques de confidentialité de sites web avec annotations sur les pratiques de données.
  - website: https://www.usableprivacy.org/data
    name: Privacy Law Corpus ("Government Privacy Instructions Corpus") (ArXiv 2022)
    description: Collection de 1 043 lois, réglementations et directives sur la vie privée couvrant 182 juridictions dans le monde.
  - website: https://www.usableprivacy.org/data
    name: Connections Between OPP-115 and the GDPR (JURIX 2020)
    description: Liens entre le schéma d'annotation OPP-115 et les principes et articles du RGPD.
  - website: https://www.usableprivacy.org/data
    name: Opt-out Choice Dataset (WWW 2020)
    description: Corpus de politiques de confidentialité pour détecter les choix de désinscription et les catégories de données.
  - website: https://www.usableprivacy.org/data
    name: Privacy Q&A Corpus (EMNLP 2019)
    description: Corpus de questions sur les politiques de confidentialité avec des annotations d'experts.
  - website: https://www.usableprivacy.org/data
    name: APP-350 Corpus (PETS 2019)
    description: Corpus de 350 politiques de confidentialité d'applications Android avec annotations sur les pratiques de confidentialité.
  - website: https://www.usableprivacy.org/data
    name: MAPS Policies Dataset (PETS 2019)
    description: URL de 441 626 politiques de confidentialité découvertes lors de l'analyse de Google Play Store.
  - website: https://www.usableprivacy.org/data
    name: Opt-out Choice Dataset (EMNLP 2017)
    description: Corpus pour identifier les choix dans les politiques de confidentialité.
  - website: https://www.usableprivacy.org/data
    name: ACL/COLING 2014 Dataset
    description: Corpus de 1 010 politiques de confidentialité des meilleurs sites web classés sur Alexa.com.
```

Reply:
So I replace the current list with this one?

Reply:
No, not the whole list, just the “Usable Privacy Policy” entry 😉